README.md

```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+# hw01
```

Comment:
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    X       |              |            |
| **Coding strategy**         |    X       |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |      X     |              |            |
| **Ease of access**          |     X      |              |            |

Overall, this is a very well-structured and organized post, and I like the way that you separate each section!